src/components/Table/TableHead/TableHead.jsx

```diff
@@ -147,9 +150,43 @@ const TableHead = ({
   closeMenuText,
   lightweight,
   i18n,
+  hasResize,
 }) => {
   const filterBarActive = activeBar === 'filter';
-
+  const utilsVar ={
+    startX: 0,
+    pressed: false,
+    index : 0
+  }
+  const [state, setState] = useState({TableHeader:[]});
+  const elementsRef = useRef(ordering.map(() => createRef()));
```

Comment:
I do not think we need to use `useRef` and `createRef()`. I think you can just map over and create like you are doing in this function passed to `useRef`